src/test/resources/config.properties

```diff
@@ -0,0 +1,12 @@
+browser=chrome
+selenium_grid_mode=false
+selenium_url=http://localhost:4444/wd/hub
+
+implicit_timeout=10
+explicit_timeout=20
+
+url=https://www.automationexercise.com/
+
+selenium_http_client_factory=apache
+chrome_driver_path=C:/Users/Olesya/chromedriver-win64/chromedriver.exe
```

Comment:
It's generally not recommended to hardcode paths like chrome_driver_path=C:/Users/Olesya/chromedriver-win64/chromedriver.exe in your project